docs/en/setup/backend/backend-storage.md

```diff
@@ -92,9 +95,19 @@ storage:
 ### Data TTL
 TTL in ElasticSearch overrides the settings of core, read [ElasticSearch section in TTL document](ttl.md#elasticsearch-6-storage-ttl)
 
-### ElasticSearch server settings
-Read the [ElasticSearch storage FAQ](../../FAQ/ES-Server-FAQ.md) if you are new to ElasticSearch. 
-And recommend read more about these configuration from ElasticSearch official document. 
+### Recommended ElasticSearch server-side configurations
+You could add following config to `elasticsearch.yml`, set the value based on your env.
+
+```yml
+# In tracing scenario, consider to set more than this at least.
+thread_pool.index.queue_size: 1000 # Only suitable for ElasticSearch 6
+thread_pool.write.queue_size: 1000 # Suitable for ElasticSearch 6 and 7
+
+# When you face query error at trace page, remember to check this.
+index.max_result_window: 1000000 # Only suitable for ElasticSearch 6. For ES 7, set `indexMaxResultWindow` under `storage`-`elasticsearch7` section in application.yml
```

Comment:
@dmsolr Is this only usable in 6.3? I remember it is not supported in ES 6.8 at least. @kezhenxu94 Could you recheck?